maintainers/maintainer-list.nix

```diff
@@ -12821,6 +12821,11 @@
     githubId = 12877905;
     name = "Roman Volosatovs";
   };
+  rxiao = {
+    email = "ben.xiao@me.com";
+    github = "benxiao";
+    githubId = 10908495;
+  };
```

Comment:
Currently a `name` attribute is required, and this is causing `ofborg` eval to fail.

Reply:
fixed